shenyu-admin/src/main/java/org/apache/shenyu/admin/config/properties/DataBaseProperties.java

```diff
@@ -35,5 +35,7 @@
 
     private String initScript;
 
+    private String customInitScript;
```

Comment:
Actually, I think it's not a good idea that introduce new field named `customInitScript` because this field is similar to `initScript`. Users may feel confused about those two fields. 
My proposal is using some special prefix. For example, if there is no prefix, means that read file from "resources"; if the path has some special prefix, like "file:/opt/xxx" means that we read from file system. 
If we want, we could support read those files from remote server, for example supporting "http://xxx/xxx/file".

But actually, it's not a good idea either.

Reply:
Can also add a enableCustomInitScript

Reply:
I am afraid not, it makes this config  more complicated. I just think that `init_script` is a common name, so we'd better not to introduce new field.

Reply:
Yes, the configuration field is still init_script, add an  enbale field, can you read the custom path

Reply:
Yes, the configuration field is still init_script, add an enable, can you read the custom path